### DISCUSSION TITLE
DER RIESE

### DISCUSSION THREAD

**Discussion by glitcheking** [nzp-team/nzportable#473] [Map Releases]

**Comment by MotoLegacy** [comment]
Hi there. In order for this to meet release standards, edit the post to make the download button bigger, and clearer. Otherwise LGTM.

**Comment by borconc** [comment]
does this map only work on PC?

**Reply by ToonLink0001** [reply]
yes i have tried

**Comment by ToonLink0001** [comment]
what are the steps for the easter egg, found 2 of the teddy bears but idk were every thing else is

**Comment by ToonLink0001** [comment]
Yo i found a bug in the map, when you crouch in the left door from the starting room you fall through the map

**Comment by SgtCookieBobble** [comment]
Great work so far! As someone that only plays on handheld, this does chug down to the low 10s with sometimes averaging in the single digitals - expected as this was made for PC in mind. One glitch I found was some weapons were missing; probably guns exclusive to the map (if there were any in Der Riese.)

Good luck with this endeavor and appreciate the work already done!

**Reply by glitcheking** [reply]
thanks i will update it in the upcoming months

**Comment by LorenzoMorelli** [comment]
Hi, thank you for your work!
Unfortuately on psvita the map doesn't render correctly, is there a way I could make it run properly on this device?

Here a screenshot of what I see from my vita:
![2024-06-23-162432-204697](https://github.com/nzp-team/nzportable/assets/24634833/454d69fe-24e2-425e-8bd2-f84946f7f89a)

**Reply by SgtCookieBobble** [reply]
I was able to fix it by disabling the global illumination in the settings (or the setting that changes the lighting, forgot what it was called exactly.)

**Reply by LorenzoMorelli** [reply]
Hi! Thank you, that made it mostly playable. It still has problems with the sky though. Do you have the same sky problem?
Moreover fps are very low in this map (around 15). Maybe it could be due to the sky issue?

This is what I have:
![2024-06-23-210205-535048](https://github.com/nzp-team/nzportable/assets/24634833/172d08f8-1a9e-4ccf-b92a-8f0cd9788968)

**Reply by SgtCookieBobble** [reply]
No, the map was made for the PC version of the game in mind; taxing anything below minimum system requirements. The sky is probably missing code that can only be found on a personal computer.

**Reply by M4x1W4lk3r** [reply]
what you did to fix it? i didn't undestood

**Comment by Phynix0027** [comment]
I did installed it on my new 3ds but it doesnt load. Why would it be?

**Comment by sealteamaqua** [comment]
glitches out and breaks.... i assume its fun tho!

**Comment by Revnova** [comment]
i don't think the person who made this is coming back to fix things. this map is not optimized at all and will not run well on anything other than pc. possibly same with other maps they've made. the creator left the old discord server in a fit calling nzp dead. so i don't believe they will be coming back.

**Reply by sealteamaqua** [reply]
damn

**Reply by ToonLink0001** [reply]
he did update it here https://github.com/nzp-team/nzportable/discussions/465

**Comment by SnugPugUnder** [comment]
Is sliding a possibility of a feature? I like to slide hop in bo3 and thought it might be fun for other people.

**Reply by MotoLegacy** [reply]
that wasnt in world at war D:

**Reply by ProfessorZIRE** [reply]
Der riese? No
The giant? Yes

**Comment by conbobcob** [comment]
i clicked the download but idk how to play it.

**Comment by leoereno** [comment]
what is the location of the 4 teddys? they appear to be invisible

**Reply by glitcheking** [reply]
Hi you have downloaden the wrong version you need der riese (updated)

**Comment by irandiaz** [comment]
There is no download link or it doesn't appear to me

**Reply by TheBetaTester000000** [reply]
wheres the download link i really wanted to try this

**Reply by glitcheking** [reply]
You neet to have the other one
https://github.com/nzp-team/nzportable/discussions/885#discussion-7091851

**Comment by patenaudejordan-creator** [comment]
<img width="452" height="253" alt="Screenshot 2026-01-16 185219" src="https://github.com/user-attachments/assets/600dbf7d-5aed-4d04-9e17-9bcede6471e6" />
every time i load up the map glicheking gives it crashes 
im on 3ds btw

**Comment by V1cboi123** [comment]
For anyone who saw this link and wondering why there is no download link is cause it been updated and moved to a different link which is this one>>>> [Der Rise Updated](https://github.com/nzp-team/nzportable/discussions/885)
